### PR TITLE
UI enhancements

### PR DESCRIPTION
**Pull request by wilspi**

### CODE REVIEW

File: templatestore/frontend/src/components/Filter/filterByJourney.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+                <div>
+                    <select id="dropdown" className={styles.filter} onChange={this.handleDropdownChange}>
+                        <option value="">Choose Journey</option>
+                        <option value="Policy">Policy</option>
+                        <option value="Claim">Claim</option>
+                        <option value="IVR">IVR</option>
+                        <option value="Purchase">Purchase</option>
+                        <option value="Pre Inspection">Pre Inspection</option>
+                        <option value="CX 360">CX 360</option>
+                        <option value="Reporting">Reporting</option>
```
Comment (wilspi) [inline templatestore/frontend/src/components/Filter/filterByJourney.js:31]: should not be hardcoded, journey should come from backend

File: templatestore/frontend/src/components/Filter/filterByLob.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+                <div>
+                    <select id="dropdown" className={styles.filter} onChange={this.handleDropdownChange}>
+                        <option value="">Choose Lob</option>
+                        <option value="Auto">Auto</option>
+                        <option value="Health">Health</option>
+                        <option value="Partnerships">Partnerships</option>
+                        <option value="Electronics">Electronics</option>
+                        <option value="Visit">Visit</option>
+                        <option value="Central">Central</option>
+                        <option value="Acko Drive">Acko Drive</option>
```
Comment (wilspi) [inline templatestore/frontend/src/components/Filter/filterByLob.js:31]: same

File: templatestore/frontend/src/pages/templateScreen.js [hunk 3]
```diff
@@ -762,7 +798,7 @@ class TemplateScreen extends Component {
                                                 );
                                             }}
                                         >
-                                            Render
+                                            Render PDF
```
Comment (wilspi) [inline templatestore/frontend/src/pages/templateScreen.js:801]: for sms case this button would show "render pdf"?

File: templatestore/frontend/src/pages/templateScreen.js [hunk 4]
```diff
@@ -1077,7 +1069,7 @@ class TemplateScreen extends Component {
                 ) : (
                     <div>
                         <div className={styles.teMarginTop20}>
-                            <label>Versions : </label>
+                            <label>Versionsss : </label>
```
Comment (wilspi) [inline templatestore/frontend/src/pages/templateScreen.js:1072]: typo

File: templatestore/frontend/src/components/Filter/filterByType.js [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+            <div>
+                <div>
+                    <select id="dropdown" className={styles.filter} onChange={this.handleDropdownChange}>
+                        <option value="">Choose Type</option>
+                        <option value="Email">Email</option>
+                        <option value="Sms">Sms</option>
+                        <option value="Policy_doc">Policy_doc</option>
+                        <option value="Whatsapp">Whatsapp</option>
+                        <option value="html_page">HTML Page</option>
+                        <option value="push_notification">Push Notification</option>
```
Comment (wilspi) [inline templatestore/frontend/src/components/Filter/filterByType.js:30]: same

**Review by wilspi — COMMENTED** [review]
enhancements for future
* pagination on home page + search 
* home page -> row clickable + open in new tab

File: templatestore/frontend/src/pages/templateScreen.js [hunk 6]
```diff
@@ -747,23 +777,44 @@ class TemplateScreen extends Component {
...
+                                                                t
+                                                            ].data,
+                                                            this.state.contextData,
+                                                            this.state.subTemplatesData[
+                                                                t
+                                                            ].renderMode
+                                                        );
+                                                    }}
+                                                >
+                                                    Render123
```
Comment (wilspi) [inline templatestore/frontend/src/pages/templateScreen.js:815]: 123?

**Review by wilspi — COMMENTED** [review]
looks good

**Review by Achintya-acko3 — APPROVED** [review]

**Pull request merged**